### PR TITLE
note to verify ssh sockets files are not stored

### PR DESCRIPTION
**Pull request by manjitkumar**
### Describe the proposed changes

Added a note to verify ssh sockets are not stored from remote machine while logging out and logging in to make add user into group effective.

### Project version

Docker Engine 1.13
### Related issue

None
### Related issue or PR in another project

None

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit 88b0c5b0e9e6104aef49d461022928b490ddef92

https://app.netlify.com/sites/docker-docs/deploys/58206f98a7e2d252499ce9d6

**Comment by johndmulhausen** [comment]
I would have the paragraph about environment bars include "...unset it" and then pivot to the other instructions

**Comment by johndmulhausen** [comment]
*vars -- sorry, on mobile :)

**Comment by manjitkumar** [comment]
@johndmulhausen I will make the suggested changes. Thank you for responding back.

File: engine/installation/linux/ubuntulinux.md [hunk 1]
```diff
@@ -292,7 +292,9 @@ To create the `docker` group and add your user:
 		Cannot connect to the Docker daemon. Is 'docker daemon' running on this host?
 
 	Check that the `DOCKER_HOST` environment variable is not set for your shell.
-	If it is, unset it.
+	If it is, unset it. Also make sure that you aren't keeping ssh socket files with a timeout. 
+	You can wait for the sockets files to be removed as per specified timeout or manually delete
```
Comment (mstanleyjones) [inline engine/installation/linux/ubuntulinux.md]: Can you include a little more info about how to make sure that you aren't keeping the ssh sockert files around? Like what do you need to look for in your SSH config or on the filesystem? 

Also consider rephrasing to something like "Verify that SSH socket files are not retained due to timeout settings."

**Comment by mstanleyjones** [comment]
I'm going to close this PR because there have been significant changes to the Ubuntu installation instructions since it was opened and because the review comments have not been addressed for 20 days.

**Pull request closed without merging**